src/components/Service.jsx

```
@@ -1,11 +1,102 @@
+        style={{ width: "100%", borderCollapse: "collapse", marginTop: 20 }}>
+          <thead>
+            <tr>
+              <th>Shipping Method</th>
+              <th>Price (VND) / KM</th>
+            </tr>
+          </thead>
+          <tbody>
+            {shippingMethod.map((item) => (
+              <>
+                <tr>
+                  <td style={{paddingLeft: "0.25em"}}>{item.shippingMethodName}</td>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)